src/Grid/FeatureGrid/FeatureGridWfs.example.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: basic.hbs
+title: FeatureGrid WFS example
+description: This example demonstrates the usage of the FeatureGrid to show data from a remote source (WFS GetFeature).
+collection: Examples
+---
+
+This example demonstrates the usage of the FeatureGrid with a remote feature source.
```

Comment:
As the example is pretty large you could add some more description.

Reply:
Fixed in 320b39b.